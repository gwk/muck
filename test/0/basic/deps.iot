{
  'cmd': '$MUCK -deps basic/index.html'
}
