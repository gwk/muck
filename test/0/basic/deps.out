
a.txt (dependents: index.html.wu module-import.txt.py):

d.txt (dependents: b.txt.py c.txt.py):
  d.txt.py:
    basic_lib.py

index.html:
  index.html.wu:
    a.txt*
    b.txt:
      b.txt.py:
        d.txt*
    c.txt:
      c.txt.py:
        c.txt.py.py
        d.txt*
    linked-src.html
    module-import.txt:
      module-import.txt.py:
        a.txt*
